Gems/Atom/RPI/Code/Source/RPI.Edit/Material/MaterialPipelineSourceData.cpp

```diff
@@ -43,6 +44,7 @@ namespace AZ
                         ->ClassElement(AZ::Edit::ClassElements::EditorData, "")
                         ->Attribute(AZ::Edit::Attributes::AutoExpand, true)
                         ->DataElement(AZ::Edit::UIHandlers::Default, &MaterialPipelineSourceData::m_shaderTemplates, "Shader Templates", "List of templates used to generate material-specific shaders.")
+                        ->DataElement(AZ::Edit::UIHandlers::Default, &MaterialPipelineSourceData::m_pipelineScript, "Pipeline Script Path", "The pipeline can use a lua script to configure shader compilation.")
```

Comment:
Later, we should update these with file picker controls. There is one that @cgalvan implemented but it requires AZ::IO::Path and one that I implemented but I have to move it from AtomTF